packages/slider/README.md

```diff
@@ -1,64 +1,124 @@
+> NOTE: The Material Web Components are distributed as ES2017 JavaScript
+> Modules, and use the Custom Elements API. They are compatible with all modern
+> browsers including Chrome, Firefox, Safari, Edge, and IE11, but an additional
+> tooling step is required to resolve *bare module specifiers*, as well as
+> transpilation and polyfills for Edge and IE11. See
+> [here](https://github.com/material-components/material-components-web-components#quick-start)
+> for detailed instructions.
 
-  1. Ensure the webcomponents polyfills are included in your HTML page
+## Example usage
 
-      - Install webcomponents polyfills
+### Continuous
 
-          ```npm i @webcomponents/webcomponentsjs```
+<img src="images/basic.png" width="140px">
 
-      - Add webcomponents polyfills to your HTML page
+```html
+<script type="module">
+  import '@material/mwc-slider';
+</script>
 
-          ```<script src="@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>```
+<mwc-slider value="25" min="10" max="50"></mwc-slider>
+```
 
-  1. Add mwc-slider to your project:
+### Discrete
 
-      ```npm i @material/mwc-slider```
+<img src="images/discrete.gif" width="163px">
 
-  1. Import the mwc-slider definition into your HTML page:
+```html
+<mwc-slider
+    step="5"
+    pin
+    markers
+    max="50"
+    value="10">
+</mwc-slider>
+```
 
-      ```<script type="module" src="@material/mwc-slider/index.js"></script>```
+### Styled
 
-      Or into your module script:
+<img src="images/styled.gif" width="188px">
 
-      ```import {Slider} from "@material/mwc-slider"```
+```html
+<style>
+  mwc-slider {
+    --mdc-theme-secondary: orange;
+    --mdc-theme-text-primary-on-dark: green;
+  }
+</style>
+<mwc-slider
+    pin
+    markers
+    max="50"
+    value="10"
+    step="5">
+</mwc-slider>
+```
 
-  1. Create an instance of mwc-slider in your HTML page, or via any framework that [supports rendering Custom Elements](https://custom-elements-everywhere.com/):
+## API
 
-      ```<mwc-slider></mwc-slider>```
 
-  1. Install the Polymer CLI:
+### Properties/Attributes
 
-      ```npm i -g polymer-cli```
+| Name      | Type      | Default | Description
+| --------- | --------- |-------- | -----------
+| `value`   | `number`  | `0`     | Current value of the slider.
+| `min`     | `number`  | `0`     | Minimum value of the slider.
+| `max`     | `number`  | `100`   | Maximum value of the slider.
+| `step`    | `number`  | `0`     | Sets the step of the slider making it discrete. Set to 0 to make continuous. Note: if `step` is greater than 0 but less than 1, it will be rounded up to 1.
+| `pin`     | `bookean` | `false` | Shows the thumb pin on a discrete slider.
+| `markers` | `bookean` | `false` | Shows the marker on the slider's track of a discrete slider.
```

Comment:
Maybe "Shows tick marks for each step on the track when the slider is discrete."